pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cognite-extractor-utils"
-version = "1.5.4"
+version = "1.5.5"
```

Comment:
There's a __version__ variable in the __init__ file that also should be updated :) 

Really wish poetry could do that automatically..

Reply:
this PR will not work for us anyway, casting bool is not safe as bool("false")= True or bool("text")=True :/ Did some search on how we can customize the casting here but couldn't find anything.